frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipeline-metrics.feature

```diff
@@ -1,4 +1,5 @@
-@pipelines
+# Tagging this feature as @broken-test, as the metrics tab will be hidden in unsupported OSP version >= 1.6.x
+@pipelines @broken-test
```

Comment:
@karthikjeeyar We will show no metrics by default anymore or just two?

Reply:
By default, we will show top 2 metrics

Reply:
Can we re-enable (and maybe update) this test then?

Reply:
updated